parts/linux/cloud-init/artifacts/ensure-no-dup.sh

```diff
@@ -7,36 +7,36 @@
 # https://github.com/kubernetes/kubernetes/pull/28717
 
 ebtables -t filter -L AKS-DEDUP-PROMISC 2>/dev/null
-if [[ $? -eq 0 ]]; then
+if [ $? = 0 ]; then
```

Comment:
we also need to take a stance on `-eq` vs. `=` for integer comparison

Reply:
I will prefer -eq too

Reply:
Just checked. -eq is for numeric comparison and = is for string. So we should use -eq for this type of comparison.